src/overlays/actors/ovl_En_Elfgrp/z_en_elfgrp.c

```diff
@@ -47,7 +78,15 @@ ActorInit En_Elfgrp_InitVars = {
     (ActorFunc)NULL,
 };
 
-void func_80A396B0(EnElfgrp* this, s32 numCutscenes) {
+/**
+ * Set the actor's cutscene to a later one, or -1 if run out.
```

Comment:
Technically it sets `this->actor.csId`, so it could be worded as
```suggestion
 * Set the actor's cutscene id to a later one, or -1 if run out.
```

Reply:
Oh that's fair. Yeah that works